### DISCUSSION TITLE
Allow us to select EarTrumpet as the flyout menu for the audio icon

### DISCUSSION THREAD

**Discussion by jrddupont** [valinet/ExplorerPatcher#3020] [Ideas]
I'd like to use [EarTrumpet](https://eartrumpet.app/) as my audio menu, but I don't want two audio icons in the system tray:
![image](https://github.com/valinet/ExplorerPatcher/assets/11879774/d474f2d0-e56d-4e03-8782-8dd7a26cda69)   
So either I'd like to be able to configure "System tray > Sound: xxxxxxxx" to be EarTrumpet, or I'd like to be able to disable the sound icon entirely

**Comment by pyrates999** [comment]
You can drag and drop the icon in the system tray to the popup menu that you don't want shown.

duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1454

Please close this one.

**Reply by jrddupont** [reply]
Ah that worked, I wasn't aware you could drag system icons around like that
